boat-maven-plugin/src/main/java/com/backbase/oss/boat/GenerateMojo.java

```diff
@@ -73,6 +77,16 @@
 @Slf4j
 public class GenerateMojo extends AbstractMojo {
 
+    private static String trimCSV(String text) {
```

Comment:
shouldn't this be at the bottom of the class?

Reply:
It's private static shouldn't be at the top?